python/ray/ml/config.py

```diff
@@ -0,0 +1,8 @@
+from typing import Dict, Any
+
+# Right now, RunConfig is just an arbitrary dict that specifies tune.run
+# kwargs.
+# TODO(xwjiang): After Tuner is implemented, define the schema
+RunConfig = Dict[str, Any]
```

Comment:
This sounds like we're still flexible about this interface (depending on what exactly mean by "schema" in this context). We have a chance to move away from our "god config" pattern here, so maybe an actual class would be nice, even if it's just a quick dataclass.

Reply:
Yep 100% agree with you! It's just out of scope for this PR and is being addressed in @xwjiang2010's Tuner interface PR.